### ISSUE TITLE
UnsupportedOperationException in some devices

### ISSUE THREAD

**Issue by hoomanmmd** [zagum/Android-SwitchIcon#9]
```
Fatal Exception: java.lang.UnsupportedOperationException
       at android.view.GLES20Canvas.clipPath(GLES20Canvas.java:413)
       at com.github.zagum.switchicon.SwitchIconView.onDraw(Unknown Source)
```
it happens on some old devices

**Comment by zagum** [comment]
Wich android os on device?

**Comment by hoomanmmd** [comment]
4.0.4 : Galaxy Tab2 7.0

**Comment by zagum** [comment]
Can you check if there is a flag in dev options on your device to force hardware acceleration?

**Comment by hoomanmmd** [comment]
unfortunately no.

**Comment by zagum** [comment]
fixed in v 1.3.7
